app/models/concerns/last_menstrual_period_measureable.rb

```diff
@@ -1,41 +1,39 @@
-# Last menstrual period calculator methods
+# Methods pertaining to last menstrual period calculation
 module LastMenstrualPeriodMeasureable
   extend ActiveSupport::Concern
 
   def last_menstrual_period_display
-    return nil unless last_menstrual_period_now
-    display_as_weeks last_menstrual_period_now
+    return nil unless _last_menstrual_period_now
+    _display_as_weeks _last_menstrual_period_now
   end
 
   def last_menstrual_period_display_short
-    return nil unless last_menstrual_period_now
+    return nil unless _last_menstrual_period_now
```

Comment:
Convention only! It more or less translates to 'treat this as private, but invoke it if absolutely necessary'. There was some fight I was having with the object that made this a good decision but I forget the finer details, can dig it up if you like.